minitask/new_import_data.py

```diff
@@ -1,13 +1,12 @@
-import wikipedia, gzip, requests
-
+import wikipedia, gzip, threading
```

Comment:
**[Required]** Still got some changes here that shouldn't be part of the PR.. might have missed these when you were reverting changes on this branch. Gonna want to revert these too.

Reply:
reverted in the latest commit

Reply:
There are still changes there though. There should not be any changes to the `new_import_data.py` file. The fact that it even appears on this PR means that there are changes. It should be exactly the same as the `main` branch.